project/build.scala

```diff
@@ -125,18 +135,17 @@ object BananaRdfBuild extends Build {
   lazy val rdfTestSuite_jvm = rdfTestSuiteM
     .project(Jvm, rdfTestSuite_common_jvm)
     .settings(Seq(resourceDirectory in Test := baseDirectory.value / "src/main/resources"):_*)
-    .dependsOn(rdf_jvm)
+    .dependsOn(rdf_jvm,ntriples_jvm)
 
   lazy val rdfTestSuite_js = rdfTestSuiteM
     .project(Js, rdfTestSuite_common_js)
     .settings(Seq(resourceDirectory in Test := baseDirectory.value / "src/main/resources"): _*)
-    .dependsOn(rdf_js)
+    .dependsOn(rdf_js,ntriples_js)
```

Comment:
Sorry, my fault. I forgot to delete all target folders.

Now I force-pushed a deletion of ntriples_js dependency and it seems to work fine.
I also tried to rebase on top of changes that were merged during the discussion of this issue